src/main/java/com/kainos/ea/model/JobRole.java

```diff
@@ -0,0 +1,47 @@
+package com.kainos.ea.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class JobRole
+{
+    private int roleID;
```

Comment:
We should try to be consistent with the variable name casing

```suggestion
    private int roleId;
```